File: LogicaDeProgramacao(JavaScript)/exercicios pag 46(Manzano)/js/exercicioC.js
/*c) Elaborar um programa que apresente no final o somatório dos valores pares existentes na faixa de
1 até 500. */

var numeroPar = 2 
var resultado = 2
var contadora = 2
var numero2 = 0


while ( contadora < 500) {

    resultado = resultado + 2
    numero2 = numeroPar + resultado
    numeroPar = numero2

    contadora = contadora + 2
}

alert("A somatoria dos valores pares até 500 é: "+numeroPar)

File: LogicaDeProgramacaoEclipse(Java)/LogicaProgramacaoJava(Eclipse)/exerciciosJava/src/manzanoPag46/ExercicioC.java
package manzanoPag46;

import java.util.Scanner;

/*c) Elaborar um programa que apresente no final o somatório dos valores pares existentes na faixa de
1 até 500. */

public class ExercicioC {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		int numeroPar = 2;
		int resultado = 2;
		int contadora = 2;
		int numero2 = 0;

		while (contadora < 500) {

			resultado = resultado + 2;
			numero2 = numeroPar + resultado;
			numeroPar = numero2;

			contadora = contadora + 2;
		}

		System.out.println("A somatoria dos valores pares até 500 é: " + numeroPar);

		sc.close();

	}

}
